doc/content/documentation/basic-types.md

```diff
@@ -141,3 +139,28 @@ Comments begin with a `#` character and continue until the end of the line. Ther
 ```phel
 # This is a comment
 ```
+
+## Arrays (deprecated, use Vectors)
```

Comment:
The idea is to remove those two deprecated data structures in the future? 🤔

Reply:
I would say yes. I would keep them in the next release but mark them as deprecated. And remove them completely in one or two versions right after, for example.

Reply:
Yes @Chemaclass is correct